### DISCUSSION TITLE
Bloatynosy 1.0

### DISCUSSION THREAD

**Discussion by Belim** [builtbybel/Bloatynosy#255] [Announcements]
## Massive Core Overhaul Launching the App to Version 1.0
**After its inception as "Bloatynosy," this release propels the app to a true milestone: version 1.0! Fun fact: Bloatynosy originally launched as my first-ever release in mid-January 2023. Since then, the app has been downloaded over 7 million times (I stopped counting earlier this year).**

Bloatynosy has been through a lot. It’s seen the highs and the lows. At its peak, it was even available in the Microsoft Store, before getting removed. The app has undergone two identity crises with name changes, somehow survived as a makeshift AI web app (Winpilot), was shut down, and then brought back to life again.

Heres what’s new in this release:
- **Bye-Bye "Bloatynosy Nue": Internally, it’s still "Bloatynosy Nue," but the app proudly wears its original name, "Bloatynosy," again in the assembly**
- Brand-new app icon!
- Modern Navigation: Redesigned tabs with a more dynamic and modern feel, expect a sleeker, more unified UI
- Tabs now come with a "close/X" button for quick cleanup
- Improved notifications area, now with a powerful search function
- All-new settings interface for easier configuration
- Upgraded Store: The store is now a primary tab, always accessible right from the main form

![explorer_FnUhf8S4NL](https://github.com/user-attachments/assets/f8b34e05-34d7-4c2a-bc21-09e4eb35b6eb)

To install a language file, simply download the language asset from [here](https://github.com/builtbybel/Bloatynosy/tree/main/languages) and move it to the directory where the .exe file is located. If you'd like to contribute translations, you can find the necessary resource files on GitHub: [Bloatynosy Locales](https://github.com/builtbybel/Bloatynosy/tree/main/src/Bloatynosy/Locales). You can also use [Hakan's translation tool](https://github.com/HakanL/resxtranslator) to assist with the process

Some of you want to thank me, so [I’ll drop my PayPal link here](https://paypal.com/donate?hosted_button_id=MY7HX4QLYR4KG). Thanks a lot, friends, and see you soon!

<details>
  <summary>If you haven’t heard yet, BloatyNosy has been brought back to life, read on to learn more...</summary>

You probably remember my classic BloatynosyApp, which had to go through some renaming and eventually got discontinued. Everything that came after, like the web-based "Winpilot," didn't last long either. Honestly, I thought I was done with Bloatynosy-well, at least with the name. But then, during a recent visit to a well-known company, I saw an employee Googling the app and trying to download it, and I had to stifle a laugh. Apparently, the legend lives on. The employee and I had a good laugh about it, and he even called it the epitome of debloating.
Sometimes, I think a little too long about things, and then a useful project just pops out. Bloatynosy was one of those projects. Sure, the name wasn't for everyone, but it was the go-to tool for cleaning up and debloating Windows. Recently, I added an OOBE assistant called "Fresh11" to the FlybyScript—basically, an all-in-one solution and I figured, why not use the same foundation to bring Bloatynosy back to life?

I had a hard time committing to it, but in the end, I went for it. Now I can proudly say that after Bloatynosy, there won’t be anything newer or better. This time, it'll be even leaner, more efficient, smaller, and faster. A simple .exe that anyone can understand, doing exactly what bugs us after installing Windows 11. **The reborn Bloatynosy carries the "Nue" extension (Bloatynosy Nue)**, not "New" ;) The last solid, classic version was 0.85, and that's where Im picking up (with a fresh new interface). The build carries a small 11.x version number.

<hr /><em>This discussion was created from the release <a href='https://github.com/builtbybel/Bloatynosy/releases/tag/1.0.0.280'>Bloatynosy 1.0</a>.</em>

**Comment by VenusGirl** [comment]
Installing language files does not apply.
